libs-scala/contextualized-logging/src/main/scala/com/digitalasset/logging/ContextualizedLogger.scala

```diff
@@ -44,35 +39,6 @@ final class ContextualizedLogger private (val withoutContext: Logger) {
   val warn = new LeveledLogger.Warn(withoutContext)
   val error = new LeveledLogger.Error(withoutContext)
 
-  private def internalOrUnknown(code: Status.Code): Boolean =
-    code == Status.Code.INTERNAL || code == Status.Code.UNKNOWN
-
-  private def logError(t: Throwable)(implicit loggingContext: LoggingContext): Unit =
-    error("Unhandled internal error", t)
-
-  def logErrorsOnCall[Out](implicit
```

Comment:
Are you sure it is not use by canton ?

Reply:
https://github.com/DACH-NY/canton/blob/683ea5f48bd56efe7dbf56546c5e15c696bb76d6/community/ledger/ledger-api-core/src/main/scala/com/digitalasset/canton/platform/apiserver/services/ApiPackageService.scala#L56

Reply:
That is a dependency on a different class with similarly named methods that re-implement the ones here (see [here](https://github.com/DACH-NY/canton/blob/961944e87aed7bdb53db5622b22481ca269958bf/community/util-logging/src/main/scala/com/digitalasset/canton/logging/TracedLoggerOps.scala)).

> Are you sure it is not use by canton ?
Reasonably so: checking out the latest `main` on the `canton` repo, checking out this commit in the `daml` submodule and running an `sbt clean Test/compile` ends without errors.